xml/System.Globalization/CultureInfo.xml

```diff
@@ -312,52 +312,10 @@
-  
-```csharp  
-  
-ResourceContext context = ResourceManager.Current.DefaultContext;  
-var list = new List<String>();  
-if (CultureInfo.CurrentCulture.Name == "en-US")  
-   list.Add("fr-FR");  
-else  
-   list.Add("en-US");  
-  
-context.Languages = list;  
-block.Text += String.Format("\nThe current culture is {0}\n",  
-                            CultureInfo.CurrentCulture);  
-block.Text += String.Format("The current culture UI is {0}\n",  
-                            CultureInfo.CurrentUICulture);  
-// The example displays output like the following if run on a system  
-// whose current culture is English (United States):  
-// The current culture is fr-FR  
-// The current UI culture is fr-FR  
-  
-```  
-  
-```  
-  
-Dim context As ResourceContext = ResourceManager.Current.DefaultContext  
-Dim list As New List(Of String)()  
-If CultureInfo.CurrentCulture.Name = "en-US" Then  
-   list.Add("fr-FR")  
-Else  
-   list.Add("en-US")  
-End If  
-context.Languages = list  
-block.Text += String.Format("{1}The current culture is {0}{1}",  
-                            CultureInfo.CurrentCulture, vbCrLf)  
-block.Text += String.Format("The current culture UI is {0}\n",  
-                            CultureInfo.CurrentUICulture)  
-' The example displays output Like the following if run on a system  
-' whose current culture Is English (United States):  
-'    The current culture Is fr-FR  
-'    The current UI culture Is fr-FR  
-  
-```  
-  
-   
+## The current culture and UWP apps  
+ In Universal Windows Platform (UWP) apps, the <xref:System.Globalization.CultureInfo.CurrentCulture%2A> and <xref:System.Globalization.CultureInfo.CurrentUICulture%2A> properties are read-write; you can use to both get and set the current culture and the current UI culture, respectively. 
+
+In .NET Framework and .NET Core apps, the current culture is a per-thread setting, and the <xref:System.Globalization.CultureInfo.CurrentCulture%2A> and <xref:System.Globalization.CultureInfo.CurrentUICulture%2A> properties reflect the culture of the current thread only. In UWP apps, however, the current culture is a per-app setting. Setting the <xref:System.Globalization.CultureInfo.CurrentCulture%2A> or <xref:System.Globalization.CultureInfo.CurrentUICulture%2A> property changes the current culture or the current UI culture for the entire app; these properties cannot be set on a per-thread basis.
```

Comment:
May be instead of saying "per-app setting" we can explain the value is mapped to Windows.ApplicationModel.Resources.Core.ResourceManager.DefaultContext.Languages which is a global setting to the app.